### PR TITLE
Issue 211

### LINKED ISSUE #211: Add Guide to PHARMACOLOGY
Adding to list of sources to retrieve `PubChem SID` in hopes of increasing DGIdb normalization

Believe we will want to use the following files:
- [ligand](https://www.guidetopharmacology.org/DATA/ligands.tsv)
- [ligand ID mapping](https://www.guidetopharmacology.org/DATA/ligand_id_mapping.tsv)

### PR DESCRIPTION
**Pull request by korikuzma**
- Closes #211 
- Fixed bugs
    - ChEMBL should be an `xref` now
    - Query tests should only use mock database if `TEST` env is set, else use local db
- CLI should allow `update_merged` flag to be set without using the `normalizer` or `update_all` flags

### CODE REVIEW

**Review by mcannon068nw — DISMISSED** [review]
I think this looks good but I would double check with @jsstevenson

**Comment by jsstevenson** [comment]
I don't think I can comment on the file directly but it looks like you renamed test_database.py to test__database.py

**Comment by korikuzma** [comment]
> I don't think I can comment on the file directly but it looks like you renamed test_database.py to test__database.py

Yes. I wanted this test to run first so I just renamed it

**Comment by jsstevenson** [comment]
> @jsstevenson Yes. I wanted this test to run first so I just renamed it

Ah, cool. I think we can probably do something like this for any tests that we want to manually override the ordering of:

```
@pytest.hookimpl(hookwrapper=True)
def pytest_collection_modifyitems(items):
    # naive solution
    first_i = 0
    for i, item in enumerate(items):
        if item.originalname == 'the test that i want to go first':
            first_i = i
            break
    items.insert(0, items.pop(first_i))
    yield items
```

**Review by mcannon068nw — APPROVED** [review]

**Review by jsstevenson — APPROVED** [review]

File: therapy/etl/guidetopharmacology.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    def _transform_ligands(self, data: dict) -> None:
+        """Transform ligands data file and add this data to `data`.
+
+        :param dict data: Transformed data
+        """
+        with open(self._ligands_file, "r") as f:
+            rows = csv.reader(f, delimiter="\t")
+            next(rows)
+
+            for row in rows:
```
[the remark this answers is not in the document]
Comment (korikuzma) [reply therapy/etl/guidetopharmacology.py:109]: I had asked @mcannon068nw this question last week and he said grab everything.

File: tests/unit/test_ncit_source.py [hunk 2]
```diff
@@ -90,6 +90,7 @@ def therapeutic_procedure():
             "Treatment",
             "TX",
             "therapeutic intervention",
+            "treatment"
```
[the remark this answers is not in the document]
Comment (korikuzma) [reply tests/unit/test_ncit_source.py:93]: Yup!

**Pull request merged**